### PR TITLE
[ES] Update `Client/File formats/osu (file format)`

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Review by agatemosu — COMMENTED** [review]
dejaría los enteros como integer

**Comment by UnknownOsu** [comment]
Prefiero dejarlo en `Entero`, siento que le da mejor coherencia que dejarlo sin traducir

**Comment by agatemosu** [comment]
entonces los String serían "Cadena"

File: wiki/Client/File_formats/osu_(file_format)/es.md [hunk 1]
```diff
@@ -171,49 +166,50 @@ Se hereda el segundo punto de tiempo a los 12 segundos, cambiando la velocidad d
...
 
-Los tipos de objetos se almacenan en un número integer de 8 bits donde cada bit es una marca con un significado especial. El tipo de objeto base viene dado por los bits 0, 1, 3 y 7 (de menos a más significativo):
-
-- 0: Círculo
-- 1: Slider
-- 3: Spinner
-- 7: Nota larga de osu!mania
-
-Los bits restantes se utilizan para distinguir nuevos combos y, opcionalmente, omitir colores de combo (comúnmente llamados «color hax»):
+El parámetro de tipo del objeto es un entero de 8 bits donde cada bit es una bandera con un significado especial.
```
Comment (agatemosu) [inline wiki/Client/File_formats/osu_(file_format)/es.md:190]:
```suggestion
El parámetro de tipo del objeto es un entero de 8 bits donde cada bit es un indicador con un significado especial.
```

File: wiki/Client/File_formats/osu_(file_format)/es.md [hunk 2]
```diff
@@ -72,8 +67,8 @@ Estas opciones solo son relevantes cuando se abren mapas en el [editor de beatma
 | `Version` | String | Nombre de la dificultad |
 | `Source` | String | Medios originales para los que se produjo la canción |
 | `Tags` | Lista de cadenas separadas por espacios | Términos de búsqueda |
-| `BeatmapID` | Integer | ID de la dificultad |
-| `BeatmapSetID` | Integer | ID del beatmap |
+| `BeatmapID` | Entero | ID de la dificultad |
+| `BeatmapSetID` | Entero | ID del beatmap |
 
 ## Dificultad
```
Comment (agatemosu) [inline wiki/Client/File_formats/osu_(file_format)/es.md:73]: `0–10` -> `0-10`

File: wiki/Client/File_formats/osu_(file_format)/es.md [hunk 3]
```diff
@@ -17,7 +12,7 @@ El siguiente contenido está separado en secciones, indicadas por títulos de se
 | :-- | :-- | :-- |
 | `[General]` | Información general sobre el beatmap | `key: value` pares |
 | `[Editor]` | Configuraciones guardadas para el editor de beatmaps | `key: value` pares |
```
Comment (agatemosu) [inline wiki/Client/File_formats/osu_(file_format)/es.md:14]: Pares `key: value`

**Review by agatemosu — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**